utils/utils.go

```diff
@@ -36,3 +39,114 @@ func ImageCache() (string, error) {
 	}
 	return CreateNotExist(filepath.Join(home, "images"))
 }
+
+func AssetsCache() (string, error) {
+	home, err := HomeDir()
+	if err != nil {
+		return "", nil
+	}
+
+	return CreateNotExist(filepath.Join(home, "assets"))
+}
+
+func MicroVmCache() (string, error) {
+	home, err := HomeDir()
+	if err != nil {
+		return "", nil
+	}
+
+	return CreateNotExist(filepath.Join(home, "microvms"))
+}
+
+func Mktemp(pattern string) (string, error) {
+	return os.MkdirTemp(os.TempDir(), pattern)
+}
+
+func IsDir(path string) bool {
+	fi, err := os.Stat(path)
+
+	return err == nil && fi.IsDir()
```

Comment:
Is this intended behaviour? I don't know much Golang but AND'ing a zero value to any value always return zero even if fi.IsDir() returns a truthy value.

if this is intended behaviour then it's fine and you can ignore this message or hopefully add an explanation why